backend/models/ImageSearch.php

```diff
@@ -30,7 +30,7 @@ public function scenarios()
      */
     public function search($params)
     {
-        $query = Image::find();
+		$query = Image::find()->where(['product_id' =>$params['id']]);
```

Comment:
What if there is no 'id' key in $params array?
Besides notice to php logs what will be the behavoir of 'where' ? where product_id = null ? or what?